crates/ruff/src/rules/flake8_boolean_trap/rules/check_positional_boolean_in_def.rs

```diff
@@ -93,11 +94,16 @@ pub(crate) fn check_positional_boolean_in_def(
         return;
     }
 
-    for arg in arguments.posonlyargs.iter().chain(arguments.args.iter()) {
-        if arg.annotation.is_none() {
+    for ArgWithDefault {
+        def,
+        default: _,
+        range: _,
+    } in chain!(&arguments.posonlyargs, &arguments.args)
```

Comment:
Should we add some custom methods to `arguments` that allow you iterating over all arguments, over all defaults, etc? It feels very repetitive to have these `chain` calls sprinkled across the code base.

Reply:
Yeah, maybe, I'm gonna punt it on now under "leave it better" (it's no worse than before).